toolkit/tools/imager/imager.go

```diff
@@ -66,13 +70,17 @@ func main() {
 	config, err := configuration.LoadWithAbsolutePaths(*configFile, *baseDirPath)
 	logger.PanicOnError(err, "Failed to load configuration file (%s) with base directory (%s)", *configFile, *baseDirPath)
 
+	// Run Preinstallation script
+	err = installutils.RunPreInstallScripts(config)
+	logger.PanicOnError(err, "Failed to preinstall scripts")
+
+	// Parse the kickstart partition file
+	err = configuration.ParseKickStartPartitionScheme(&config, kickstartPartitionFile)
```

Comment:
Do we want a flag to explicitly enable this functionality for config, or is the presence of the file and lack of other settings enough?

Reply:
We could have an option in the partitions area of the config, and the validator could check that if that flag is set, there are no partitions set in the config.

Reply:
I initially considered this option, but my concern is that the users might forget to set this flag when they translate their kickstart files to image config json since this option is not supported in kickstart. Therefore, I would like to let the installer handle the decision of perform the parsing based on whether the partitioning information already exists in the image config json.